site_scons/community/linux.py

```diff
@@ -104,9 +104,11 @@ def create( self, opts ):
     #
     # Configures all the necessary environment variables for Linux
     def configure(self, env ):
-        env.Append( CPPDEFINES = ['HAVE_CONFIG_H','_GNU_SOURCE'] )
+        env.Append( CPPDEFINES = ['_GNU_SOURCE'] )
```

Comment:
I actually left those in on purpose to make it easy to switch between making warnings fatal or not, though I can take them out.